src/cuda/gpu_getxc.h

```diff
@@ -510,7 +515,13 @@ __global__ void cshell_getxcgrad_kernel()
       }
 #endif
 
+#ifdef CEW
+      devSim_dft.exc[gid] = _tmp + (dfdr_cew * (density+densityb));
```

Comment:
I think somewhere in the output we have to state / warn users that when CEw is used, then the XC energy contains both the XC energy and CEw reciprocal space potential energy (I still have to write down the equations properly to make sense and give appropriate names to the different terms).

Reply:
Fixed.